opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchPhraseQueryTest.java

```diff
@@ -104,9 +106,175 @@ public void test_zero_terms_query_parameter_lower_case() {
     Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(arguments)));
   }
 
+  @Test
+  public void test_SyntaxCheckException_when_no_arguments_match_phrase_syntax() {
+    List<Expression> arguments = List.of();
+    assertThrows(SyntaxCheckException.class,
+        () -> matchPhraseQuery.build(new MatchPhraseExpression(
+            arguments, MatchPhraseQueryTest.this.matchPhraseWithUnderscoreName)));
```

Comment:
```suggestion
            arguments, matchPhraseWithUnderscoreName)));
```

isn't that sufficient?

Reply:
This has been fixed